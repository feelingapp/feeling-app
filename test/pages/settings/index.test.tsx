import { shallow } from "enzyme"
import Settings from "../../../src/pages/settings"

describe("Settings: ", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Settings/>)
    expect(wrapper).toMatchSnapshot()
  })
})
