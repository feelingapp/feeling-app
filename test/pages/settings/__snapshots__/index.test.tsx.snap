// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings:  matches snapshot 1`] = `
<Fragment>
  <Head>
    <title>
      Settings / Feeling
    </title>
  </Head>
  <TopBar
    backButton={true}
    title="Settings"
  />
  <Main>
    <SettingItem
      href="/settings/reminders"
      subtitle="Be reminded everyday to log how you're feeling throughout the day"
      title="Reminders"
    />
    <SettingItem
      href="/settings/analytics"
      subtitle="Probabo, inquit, modo dixi, constituto, ut alterum esse admonere interesse enim ad id."
      title="Smart Analytics"
    />
    <Separator />
    <SettingItem
      href=""
      title="Terms and Conditions"
    />
    <SettingItem
      href=""
      title="Privacy Policy"
    />
    <Separator />
    <SettingItem
      subtitle="1.0"
      title="Version"
    />
    <SettingItem
      href="/signout"
      subtitle="You are signed in as Michael Lee"
      title="Sign Out"
    />
  </Main>
</Fragment>
`;
